src/types/errors/SidecarUserError.ts

```diff
@@ -0,0 +1,13 @@
+import SidecarError from "./SidecarError";
+
+export default class SidecarUserError implements SidecarError {
+  name: 'SidecarUserError';
+  message: string;
+  stack: undefined;
```

Comment:
I think there's a confusion between type and value. Here it means that `stack` is of type `undefined`, which means it can only take `undeinfed` as value.

Maybe you meant `stack: string | undefined = undefined`?